### DISCUSSION TITLE
非常奇怪的漏识别问题。

### DISCUSSION THREAD

**Discussion by mingliao1994** [PaddlePaddle/PaddleOCR#13565] [Q&A]
这里有一份还算是清晰的报告，使用PaddleOCR识别漏了整整四行，不知道为什么
![QQ_1722497019684](https://github.com/user-attachments/assets/5f980085-2e1d-4d95-bff6-bcfe024a101a)

**Comment by SWHL** [comment]
影响因素有很多，猜测可能是漏检部分纸张有轻微褶皱导致的

**Reply by mingliao1994** [reply]
之前有过不少褶皱更厉害的都不存在这个问题，就这张相当特殊。原理不明...

**Reply by UserWangZz** [reply]
降低过滤阈值看看能不能检到

**Comment by BeyondYourself** [comment]
这一看就是设置参数的问题，max length设置大一点
